logstash-output-stdout.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-stdout'
-  s.version         = '3.0.2'
+  s.version         = '4.0.0'
```

Comment:
Why major bump? This is backward compatible with previous version right? Maybe `3.1.0`?